src/config/config.php

```diff
@@ -90,6 +90,11 @@
         'composer' => \Folklore\GraphQL\View\GraphiQLComposer::class,
     ],
 
+    /*
+     * Whether or not to fire the 'TypeAdded' event. You can set this to false if you don't use it in your application.
+     */
+    'fire_type_added_events' => false,
```

Comment:
The default behaviour shoulnd't change, so this should be `true`

Reply:
Agreed, this was a mistake on my part to submit this defaulted to false.